src/main/java/randoop/operation/TypedClassOperation.java

```diff
@@ -174,4 +174,32 @@ public boolean hasWildcardTypes() {
     }
     return new ArrayList<>(paramSet);
   }
+
+  /**
+   * Returns the raw type signature for this operation. This signature consists of the classname as
+   * a fully-qualified raw type, the method name, and the argument types as fully-qualified raw
+   * types.
+   *
+   * <p>The raw type signature for a constructor <code>C()</code> has has this form instead of the
```

Comment:
It's unclear what "this form is".  Is a constructor name the name of the class?  It would be good to be explicit.